oligo_designer_toolsuite/oligo_specificity_filter/_filter_blastn.py

```diff
@@ -60,18 +76,28 @@ def __init__(
             "query_start",
             "query_end",
             "query_length",
-        ],
-    ):
+            "query_sequence",
+            "reference_start",
+            "reference_end",
+            "reference_sequence",
+            "reference_strand"
+        ], #TODO: update based on ai filters
+        ai_filter: str = None,
+        ai_filter_threshold: float = None,
+        ai_filter_path: str = None,
+):
         """Constructor for the BlastNFilter class."""
-        super().__init__(dir_output)
+        super().__init__(dir_output, ai_filter, ai_filter_threshold, ai_filter_path)
         self.blast_search_parameters = blast_search_parameters
         self.blast_hit_parameters = blast_hit_parameters
         self.names_search_output = names_search_output
 
         # Define default output format for blast search filter. The fields are:
         # query, reference, alignment_length, query_start, query_end, query_length
         if "outfmt" not in self.blast_search_parameters.keys():
-            self.blast_search_parameters["outfmt"] = "6 qseqid sseqid length qstart qend qlen"
+            self.blast_search_parameters["outfmt"] = (
+                "6 qseqid sseqid length qstart qend qlen qseq sstart send sseq sstrand"# TODO: update based on the requirements for ai filters,
+            )
```

Comment:
the output format should not be changed per default. The default should be the minimal needed for pure blast filter. As written above this should be defined in the AI filter class and then be updated here os ai_filters is not none